modules/sdk/project-templates/project-templates-soy-portlet/src/main/resources/archetype-resources/pom.xml

```diff
@@ -69,88 +69,71 @@
+                            <modulePath>${node.modules.dir}/metal-cli/index.js</modulePath>
+                            <bundleFileName></bundleFileName>
+                            <dest>${project.build.outputDirectory}/META-INF/resources</dest>
+                            <format>amd</format>
+                            <globalName></globalName>
+                            <moduleName></moduleName>
+                            <soyDeps>
+                                <soyDep>${node.modules.dir}/lexicon*/src/**/*.soy</soyDep>
+                                <soyDep>${node.modules.dir}/metal*/src/**/*.soy</soyDep>
+                            </soyDeps>
+                            <soyDest>${project.build.outputDirectory}/META-INF/resources</soyDest>
+                            <soySrcs>
+                                <soySrc>**/*.soy</soySrc>
+                            </soySrcs>
+                            <srcs>
+                                <src>**/*.es.js</src>
+                                <src>**/*.soy.js</src>
+                            </srcs>
+                        </configuration>
 					</execution>
 					<execution>
-						<id>config-js-modules</id>
+                        <id>config-js-modules</id>
 						<phase>process-classes</phase>
 						<goals>
-							<goal>exec</goal>
+							<goal>config-js-modules</goal>
 						</goals>
-						<configuration>
-							<executable>${project.build.directory}/node/node</executable>
-							<workingDirectory>${project.build.outputDirectory}/META-INF/resources</workingDirectory>
-							<arguments>
-								<argument>${node.modules.dir}/liferay-module-config-generator/bin/index.js</argument>
-								<argument>--config=</argument>
-								<argument>--extension=</argument>
-								<argument>--filePattern</argument>
-								<argument>**/resources/*.js</argument>
-								<argument>--format</argument>
-								<argument>/_/g,-</argument>
-								<argument>--ignorePath</argument>
-								<argument>true</argument>
-								<argument>--moduleConfig</argument>
-								<argument>${project.basedir}/package.json</argument>
-								<argument>--namespace</argument>
-								<argument>Liferay.Loader</argument>
-								<argument>--output</argument>
-								<argument>${project.build.outputDirectory}/META-INF/config.json</argument>
-								<argument>--moduleRoot</argument>
-								<argument>${project.build.outputDirectory}/META-INF/resources</argument>
-								<argument>${project.build.outputDirectory}/META-INF</argument>
-							</arguments>
-						</configuration>
+                        <configuration>
+                            <nodePath>${project.build.directory}/node/node</nodePath>
+                            <workingPath>${project.build.outputDirectory}/META-INF/resources</workingPath>
+                            <modulePath>${node.modules.dir}/liferay-module-config-generator/bin/index.js</modulePath>
+                            <config></config>
```

Comment:
also set defaults for these and remove them from configuration
`config`
`extension`
`format`
`ignorePath`
`namespace`